platform/admin/core/admin-core-migration-commands/src/test/java/org/codice/ddf/migration/commands/ImportCommandTest.java

```diff
@@ -35,10 +38,26 @@ public void testConstructor() throws Exception {
   }
 
   @Test
-  public void testExecuteWithSubject() throws Exception {
+  public void testExecuteWithSubjectAndProfileDisabled() throws Exception {
     command.executeWithSubject();
 
-    Mockito.verify(service).doImport(Mockito.eq(exportedPath), Mockito.notNull());
+    Mockito.verify(service)
+        .doImport(
+            Mockito.eq(exportedPath), Mockito.same(Collections.emptySet()), Mockito.notNull());
     verifyPostedEvent("import");
   }
+
+  @Test
+  public void testExecuteWithSubjectAndProfileEnabled() throws Exception {
+    initCommand(new ImportCommand(service, security, eventAdmin, session, true));
+
+    command.executeWithSubject();
+
+    final ArgumentCaptor<Set<String>> mandatoryMigratables = ArgumentCaptor.forClass(Set.class);
+
+    Mockito.verify(service)
+        .doImport(Mockito.eq(exportedPath), mandatoryMigratables.capture(), Mockito.notNull());
+    verifyPostedEvent("import");
+    Assert.assertThat(mandatoryMigratables.getValue(), Matchers.contains("ddf.profile"));
```

Comment:
I think this should be hamcrest `assertThat` 
https://codice.atlassian.net/wiki/spaces/DDF/pages/6291523/Unit+Testing

Reply:
The guideline says to use `assertThat()` with hamcrest matchers which is what I am doing.

Reply:
`Favor assertThat()and hamcrest Matchers or AssertJ over plain jUnit assert methods`

Reply:
Yeah, favor `assertThat()` which is part of JUnit in order to use hamcrest matchers over plain JUnit assert methods such as `assertEquals()`

Reply:
`Assert.assertThat` is a JUnit assert method.  I think the goal of this sentence is to convey that we should be using Hamcrest `assertThat` over JUnit `assertThat`

Reply:
I read this as _use assertThat() with hamcrest matchers_ and not _use hamcrest assertThat() and matchers_. 
Further more, developers will typically expect that JUnit be used to assert stuff and hamcrest be used to match stuff.
Finally, the `MatcherAssert` class is not even documented as it was not really meant to be part of the public API but is public only so that `Assert.assertThat()` can call it.

Reply:
The guideline was updated by @lessarderic to clarify we should use `Assert.assertThat()` with hamcrest matchers as done here 😉